js/ui/list/ui.list.edit.decorator.selection.js

```diff
@@ -49,19 +49,20 @@ registerDecorator(
             const $itemElement = config.$itemElement;
             const $container = config.$container;
 
-            const $control = $('<div>').addClass(this._controlClass);
-            new this._controlWidget($control, extend(this._commonOptions(), {
+            new this._controlWidget($('<div>').addClass(this._controlClass).appendTo($container), extend(this._commonOptions(), {
                 value: this._isSelected($itemElement),
                 focusStateEnabled: false,
                 hoverStateEnabled: false,
                 onValueChanged: (function(e) {
                     this._processCheckedState($itemElement, e.value);
                     e.event && e.event.stopPropagation();
-                }).bind(this)
+                }).bind(this),
+                elementAttr: {
+                    class: this._controlClass
```

Comment:
What's it done for?

Reply:
It's generator bug WA. I don't wanna remove it on the off-chance, to make it easier to find it in the future. Can it break smthing, how do u think?

Reply:
I don't think so. Just looks weird :)
Could you add a comment note to remember to remove this WA in the future?

Reply:
I decided to backup this WA locally and not push to the repo :-) It's will be safer